src/backup.ts

```diff
@@ -179,13 +232,55 @@ class Backup {
    * // Await completion of the backup operation.
    * await backupOperation.promise();
    */
+  /**
+   * Copy a backup.
+   *
+   * @method Backup#create
+   * @param {CopyBackupOptions} options Parameters for copying a backup.
+   * @param {CallOptions} [options.gaxOptions] The request configuration
+   *     options, See {@link https://googleapis.dev/nodejs/google-gax/latest/interfaces/CallOptions.html|CallOptions}
+   *     for more details.
+   * @param {CopyBackupCallback} [callback] Callback function.
+   * @returns {Promise<CopyBackupResponse>} When resolved, the copy backup
+   *     operation will have started, but will not have necessarily completed.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const oneDay = 1000 * 60 * 60 * 24;
+   * const expireTime = Date.now() + oneDay;
+   * const versionTime = Date.now() - oneDay;
+   * const sourceBackup = instance.backup('my-source-backup');
+   * const [, sourceBackupOperation] = await sourceBackup.create({
+   *   databasePath: 'projects/my-project/instances/my-instance/databases/my-database',
+   *   expireTime: expireTime,
+   *   versionTime: versionTime,
+   *   encryptionConfig: {
+   *     encryptionType: 'CUSTOMER_MANAGED_ENCRYPTION',
+   *     kmsKeyName: 'projects/my-project-id/my-region/keyRings/my-key-ring/cryptoKeys/my-key',
+   *   },
+   * });
+   * // Await completion of the source backup operation.
+   * await sourceBackupOperation.promise();
+   * const copyBackup = instance.backup('my-copy-backup', sourceBackup.formattedName_);
```

Comment:
I think this should be:
```suggestion
   * const copyBackup = instance.copy_backup('my-copy-backup', sourceBackup.formattedName_);
```

But see also my comment on the `copy_backup` name below.